package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "nodejs-mentoring",
+  "version": "1.0.0",
+  "description": "EPAM node.js mentoring Excercises and tasks ",
+  "type": "module",
+  "scripts": {
+    "start": "node -v && node main.js",
+    "dev": " node main.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/attilagal-epam/nodejs-mentoring.git"
+  },
+  "author": "Attila Barnabas Gal",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/attilagal-epam/nodejs-mentoring/issues"
+  },
+  "homepage": "https://github.com/attilagal-epam/nodejs-mentoring#readme",
+  "devDependencies": {
+    "nodemon": "^3.1.4"
```

Comment:
TODO: make use of this dependency, see comments on `scripts:dev` section

Reply:
See previous [comment](https://github.com/attilagal-epam/nodejs-mentoring/pull/1/files#r1725210680).
I Only left it in devDependencies because of AC "[nodemon](https://github.com/remy/nodemon) is installed as dev dependency."

Reply:
Yes - because it supposed to be referred from the scripts :)